RiotSharpTest/ApiTestHelper.cs

```diff
@@ -0,0 +1,190 @@
+            FreeToPlay = false,
+            Active = false,
+            BotEnabled = false,
+            BotMmEnabled = false,
+            RankedPlayEnabled = false
+
+        };
+
+        private static Mastery mast1 = new Mastery()
+        {
+            Id = 12345,
+            Rank = 1
+        };
+
+        private static Mastery mast2 = new Mastery()
+        {
+            Id = 54321,
+            Rank = 2
+        };
+
+        private static MasteryPage mp1 = new MasteryPage()
+        {
+            Current = true,
+            Id = 1234,
+            Masteries = new List<Mastery>()
+            {
+                mast1,
+                mast2
+            },
+            Name = "Test"
+        };
+
+        private static MasteryPages pg1 = new MasteryPages()
+        {
+            Pages = new List<MasteryPage>()
+            {
+                mp1
+            },
+            SummonerId = 57039082
+        };
+
+        private static List<MasteryPages> masteryList = new List<MasteryPages>()
+        {
+
+        };
+
+
+        private static string team = "TEAM-66fbf960-a74c-11e4-a5a6-c81f66dd45c9";
+        private static string team2 = "TEAM-0a2ea1b0-f1cf-11e4-bc7e-c81f66ddabda";
+        private static int gameId = 1826202304;
+        private static RiotApi api;
+        private static Queue queue = Queue.RankedSolo5x5;
+        private static Region region = Region.na;
+        private static RiotSharp.MatchEndpoint.Enums.Season season = RiotSharp.MatchEndpoint.Enums.Season.Season2015;
+        private static DateTime beginTime = new DateTime(2015, 01, 01);
+        private static DateTime endTime { get { return DateTime.Now; } }
+
+        public static Region GetRegion()
+        {
+            return region;
+        }
+
+        public static Summoner GetSummoner()
+        {
+            return sum1;
+        }
+
+        public static List<Summoner> GetSummonerList()
+        {
+            return new List<Summoner>()
+            {
+                sum1,
+                sum2
+            };
+        }
+
+        public static Champion GetChampion()
+        {
+            return chp1;
+        }
+
+        public static List<Champion> GetChampionList()
+        {
+            return new List<Champion>()
+            {
+                chp1,
+                chp2
+            };
+        }
+
+        public static IRateLimitedRequester GenerateRequester(string json)
+        {
+            IRateLimitedRequester req = null;
+            var mockReq = new Mock<IRateLimitedRequester>();
+            mockReq.Setup(x => x.CreateGetRequest(It.IsAny<string>(), It.IsAny<Region>(),
+                                    It.IsAny<List<string>>(), It.IsAny<bool>())).Returns(json);
+            mockReq.Setup(x => x.CreateGetRequestAsync(It.IsAny<string>(), It.IsAny<Region>(),
+                        It.IsAny<List<string>>(), It.IsAny<bool>())).ReturnsAsync(json);
+            req = mockReq.Object;
+            return req;
```

Comment:
These two lines can be done inline.  return mockReq.Object;

Reply:
@dresdor this was closed in favor of #380 and #383, no need for a review 👍